src/SonarQube.Client/Services/SonarQubeClient.cs

```diff
@@ -89,68 +107,67 @@ public Task<Result<ServerIssue[]>> GetIssuesAsync(ConnectionRequest connection,
                 });
         }
 
-        public Task<Result<OrganizationResponse[]>> GetOrganizationsAsync(ConnectionRequest connection,
-            OrganizationRequest request, CancellationToken token)
+        public Task<Result<OrganizationResponse[]>> GetOrganizationsAsync(OrganizationRequest request, CancellationToken token)
         {
             const string OrganizationsAPI = "api/organizations/search"; // Since 6.2; internal
 
             var query = AppendQueryString(OrganizationsAPI, "?p={0}&ps={1}", request.Page, request.PageSize); // TODO: should handle optional params
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)